### PR TITLE
Random tweaks

### PR DESCRIPTION
**Pull request by georgecrawford**
A few distinct improvements I've added, and I think are useful.

### CODE REVIEW

File: lib/adapter.js [hunk 1]
```diff
@@ -166,8 +170,6 @@ adapter.emit = function (event, runner, cid, capabilities, payload, err) {
 
     if (err) {
         runner.lastError = err
-    } else if (event === 'test:start') {
-        delete runner.lastError
```
Comment (christian-bromann) [inline lib/adapter.js]: is this not necessary after all?

Comment (georgecrawford) [inline lib/adapter.js]: Here instead: https://github.com/webdriverio/wdio-mocha-framework/pull/4/files#diff-a7bcb5fcc46018435e1ce3fa65fcb992R56

Comment (christian-bromann) [inline lib/adapter.js]: ohh right

**Comment by christian-bromann** [comment]
LGTM

**Pull request merged**